Cargo.toml

```diff
@@ -37,8 +37,6 @@ windows = { version = ">=0.58,<0.60", features = [
 [build-dependencies]
 cfg_aliases = { version = "0.2", default-features = false }
 mozbuild = { version = "0.1", default-features = false, optional = true }
-
-[target.'cfg(not(windows))'.build-dependencies]
 # Don't increase beyond what Firefox is currently using: https://searchfox.org/mozilla-central/source/Cargo.lock
 bindgen = { version = "0.69", default-features = false, features = ["runtime"] }
```

Comment:
My understanding here is, that `cfg(not(windows))` is the target OS of the `build.rs` binary. We might be compiling Firefox for Windows on a Linux machine. Thus this `cfg` is removed.